data/migrations/20211108175854_meetings.js

```diff
@@ -2,28 +2,31 @@ exports.up = function (knex) {
   return knex.schema
     .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
     .createTable('meetings', function (table) {
-      table.increments('meeting_id').notNullable().unique().primary();
-      table.timestamps(true, true);
+      table.string('meeting_id').notNullable().unique().primary();
       table.string('meeting_topic').notNullable();
-      table.integer('meeting_start_date').notNullable().unsigned();
-      table.integer('meeting_end_date').notNullable().unsigned();
+      table.timestamps(true, true);
+      table.string('meeting_start_time').notNullable().unsigned();
```

Comment:
This refactoring from date to time and host/attendee to mentor/mentee brings a lot more semantic meaning to these fields! A welcome improvement 🙌🏾